tests/unit/AgreementType.spec.ts

```diff
@@ -9,17 +9,18 @@ const summaryErrorMessageSelector: string = '.agreement-invalid-message'
 const summaryTextSelector: string = '.summary-desc'
 
 const agreementTypeTestCases = [
-  {
-    entityType: CorpTypeCd.COOP,
-    sampleAgreement: {
-      agreementType: 'sample'
-    },
-    customAgreement: {
-      agreementType: 'custom'
-    },
-    sampleSummaryText: '',
-    customSummaryText: ''
-  },
+  // *** TODO: fix
+  // {
+  //   entityType: CorpTypeCd.COOP,
+  //   sampleAgreement: {
+  //     agreementType: 'sample'
+  //   },
+  //   customAgreement: {
+  //     agreementType: 'custom'
+  //   },
+  //   sampleSummaryText: '',
+  //   customSummaryText: ''
+  // },
```

Comment:
O right, if Coops don't have agreement type, we could probably just remove this entity type from the tests maybe?

Reply:
Haha, thanks for the solution :)